libraries/AsyncUDP/library.json

```diff
@@ -1,17 +1,15 @@
 {
-  "name":"AsyncUDP",
-  "description":"Asynchronous UDP Library for ESP32",
-  "keywords":"async,udp,server,client,multicast,broadcast",
-  "authors":
-  {
+  "name": "AsyncUDP",
+  "description": "Asynchronous UDP Library for ESP32",
+  "keywords": "async,udp,server,client,multicast,broadcast",
+  "authors": {
     "name": "Hristo Gochkov",
     "maintainer": true
   },
-  "repository":
-  {
-    "type": "git",
-    "url": "https://github.com/me-no-dev/ESPAsyncUDP.git"
-  },
-  "frameworks": "arduino",
-  "platforms":"espressif"
-}
+  "frameworks": [
+    "arduino"
+  ],
+  "platforms": [
+    "espressif32"
```

Comment:
by changing this to espressif32 you are desupporting the esp8266, you will need to add "espressif" as a second element in this list.